### PR TITLE
fix(ui): TE-1051 renamed cohort-detector route to be consistent with page title

### PR DESCRIPTION
**Pull request by alcatraz627**
#### Issue(s)

- [TE-1051](https://cortexdata.atlassian.net/browse/TE-1051)

#### Description

Change routing name `/cohort-detector` to `/cohort-recommender` to be more consistent with the page title

### CODE REVIEW

File: thirdeye-ui/src/app/utils/routes/routes.util.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ const PLACEHOLDER_ROUTE_ID = ":id";
 
 export const AppRouteRelative = {
     ADMIN: "admin",
-    COHORT_DETECTOR: "cohort-detector",
+    COHORT_DETECTOR: "cohort-recommender",
```
Comment (spham92) [inline thirdeye-ui/src/app/utils/routes/routes.util.ts:26]: Rename this from `COHORT_DETECTOR` => `COHORT_RECOMMENDER` and ensure all usages of this is refactored as well

Comment (alcatraz627) [reply thirdeye-ui/src/app/utils/routes/routes.util.ts]: Updated.

File: thirdeye-ui/src/app/components/admin-page/recent-failures/recent-failures.component.tsx [hunk 2]
```diff
@@ -12,31 +12,110 @@
...
+            Number(searchParams.get(TimeRangeQueryStringKey.END_TIME)),
+        ],
+        [searchParams]
+    );
+
+    const {
+        timeRangeDuration,
+        recentCustomTimeRangeDurations,
+        setTimeRangeDuration,
+    } = useTimeRange();
```
Comment (spham92) [inline thirdeye-ui/src/app/components/admin-page/recent-failures/recent-failures.component.tsx:71]: Don't use `useTimeRange `. I've been trying to move off this and strictly use whats in the search params.

Comment (alcatraz627) [reply thirdeye-ui/src/app/components/admin-page/recent-failures/recent-failures.component.tsx]: I've removed the TE-1002 (date filter in admin) code to [#774](https://github.com/startreedata/thirdeye/pull/774), will address this comment there

**Pull request closed without merging**